### PR TITLE
Align strategies.yaml load logging with other config files

### PR DESCRIPTION
**Pull request by randall**
Following the convention used in 9f5396ee87c48e645ffc712f18de1d32849341dd, change the strategies.yaml loading log messages to match.

Empty file:
```
[Aug 18 16:41:05.193]  NOTE: strategies.yaml loading ...
[Aug 18 16:41:05.195]  NOTE: No NextHop strategy configs were loaded.
[Aug 18 16:41:05.195]  NOTE: strategies.yaml finished loading
```

Non-existent file:
```
[Aug 19 09:00:08.649]  NOTE: strategies.yaml loading ...
[Aug 19 09:00:08.649]  NOTE: strategies.yaml doesn't exist
[Aug 19 09:00:08.649]  NOTE: strategies.yaml finished loading
```

Various errors:
```
[Aug 18 16:43:22.022]  NOTE: strategies.yaml loading ...
[Aug 18 16:43:22.023]  ERROR: yaml-cpp: error at line 4, column 13: illegal map value
[Aug 18 16:43:22.023]  ERROR: strategies.yaml failed to load
```

```
[Aug 18 16:42:07.719]  NOTE: strategies.yaml loading ...
[Aug 18 16:42:07.719]  ERROR: malformed strategies.yaml file, expected a 'strategies' sequence
[Aug 18 16:42:07.719]  ERROR: strategies.yaml failed to load
```

### CODE REVIEW

File: proxy/http/remap/NextHopStrategyFactory.cc [hunk 1]
```diff
@@ -220,9 +236,6 @@ NextHopStrategyFactory::loadConfigFile(const std::string &fileName, std::strings
         std::ifstream file(fileName + "/" + i.data());
         if (file.is_open()) {
           while (std::getline(file, line)) {
-            if (line[0] == '#') {
-              // continue;
```
Comment (jrushford) [inline proxy/http/remap/NextHopStrategyFactory.cc:224]: @randall Was this an issue, skipping comment lines?

Comment (randall) [reply proxy/http/remap/NextHopStrategyFactory.cc:224]: I removed it because the continue wall was commented out. It looked like it was nothing.

Comment (randall) [reply proxy/http/remap/NextHopStrategyFactory.cc]: I reverted that part of the commit just in case you want to add back comment support.

**Review by jrushford — APPROVED** [review]
lgtm

**Comment by zwoop** [comment]
This has some serious dependencies on things which are not in 9.1.x branch (they are marked for 9.2.x). If we really need this, either make a new PR just with these changes, or we have to decide to put all those other parent selection refactoring etc. into 9.1.x I think.

Moving this out to 9.2.x for now.

**Pull request merged**